crates/python/src/qpu/client.rs

```diff
@@ -185,12 +184,8 @@ impl PyQcsClient {
 
     #[staticmethod]
     #[args("/", profile_name = "None", use_gateway = "None")]
-    pub fn load(
-        py: Python<'_>,
-        profile_name: Option<String>,
-        use_gateway: Option<bool>,
-    ) -> PyResult<&PyAny> {
-        future_into_py(py, async move {
+    pub fn load(profile_name: Option<String>, use_gateway: Option<bool>) -> PyResult<Self> {
+        crate::utils::py_sync!(async move {
```

Comment:
The pattern I'd suggest, while we're here: a `mod blocking` like `reqwest` [does](https://docs.rs/reqwest/latest/reqwest/blocking/index.html) . Could also be a crate feature in the same way.

Then the primary `async` methods can still be available. Because, in principle, `async` makes sense here for I/O blocking calls, and those should be accessible from the get-go in these bindings, even if `pyquil` doesn't actually use them yet. Don't _make_ the user use the sync wrapper, right?

Reply:
By crate feature do you mean like a python "[extra](https://peps.python.org/pep-0508/#extras)"?

Since a lot of the code would be duplicated between a `blocking` and "standard" module and the underlying function implementations are identical, I was thinking something like the following. 

```rs
impl PyQcsClient {
    async fn load(profile_name: Option<String>, use_gateway: Option<bool>) -> PyResult<Self> {
        Ok(Self(/* ... */))
    }
}

#[pymethods]
impl PyQcsClient {
    // ...

    #[staticmethod]
    #[args("/", profile_name = "None", use_gateway = "None")]
    #[pyo3(name = "load")]
    pub fn py_load(profile_name: Option<String>, use_gateway: Option<bool>) -> PyResult<Self> {
        py_sync!(Self::load(profile_name, use_gateway))
    }

    #[staticmethod]
    #[args("/", profile_name = "None", use_gateway = "None")]
    #[pyo3(name = "load_async")]
    pub fn py_load_async(
        py: Python<'_>,
        profile_name: Option<String>,
        use_gateway: Option<bool>,
    ) -> PyResult<&PyAny> {
        py_async!(py, Self::load(profile_name, use_gateway))
    }
}
```

Reply:
Ah - yeah the way you have it here should be good. Idea with the crate feature was to reduce compile time / binary size, but since we're distributing as a wheel in most cases that does no good. That might be better as an added feature in the SDK itself.